packages/common-all/src/types/typesv2.ts

```diff
@@ -24,7 +24,7 @@ export type EngineDeleteOpts = {
    */
   metaOnly?: boolean;
   /**
-   * If node is deleted and parents are stubs, default behavior is to alsod elete parents
+   * If node is deleted and parents are stubs, default behavior is to alsod delete parents
```

Comment:
nit: lol, typo on alsod

Reply:
haha, can't have you catching all of them :)